tfhe/src/boolean/parameters/mod.rs

```diff
@@ -182,8 +182,8 @@ pub const TFHE_LIB_PARAMETERS: BooleanParameters = BooleanParameters {
     lwe_dimension: LweDimension(630),
     glwe_dimension: GlweDimension(1),
     polynomial_size: PolynomialSize(1024),
-    lwe_modular_std_dev: StandardDev(0.000030517578125),
```

Comment:
Don't know if we want that yet, the std dev are generated by optimizers and not formatted this way, will be super annoying to manage manially, I would skip this lint for now

Reply:
The lint has an automatic fix so it shouldn't be too annoying
But I agree it's not really useful in out case

Reply:
Maybe we can allow it for parameter modules

Reply:
I'll keep it disabled it for now